dapp/src/store/mobx/state/dashboard.js

```diff
@@ -445,10 +457,81 @@ export default class Dashboard {
     this.isCommunityMember = memberData?.data?.communityEntities?.length > 0
   })
 
-
   get addedPlugins () {
     return Object.keys(
       pickBy(this?.plugins, pluginKey => !pluginKey?.isRemoved)
     ).sort()
   }
+
+  get bridgeStatus () {
+    if (this?.community?.bridgeDirection === 'home-to-foreign') {
+      if (this.rootStore.network.networkId === 122) {
+        return {
+          from: {
+            network: this.rootStore.network.homeNetwork,
+            bridge: 'foreign',
+            balanceKey: 'fuse'
```

Comment:
Can we add more data here, like web3 instance and tokenAddress?
 I would love to use less `balanceKey`, and have most of the data given.